### DISCUSSION TITLE
Weten in Wall Display smartHome switch

### DISCUSSION THREAD

**Discussion by michaelschort** [arendst/Tasmota#10326] [Q&A] [answered]
I Just saw that cool new smartHome switch product Family by a company called weten. It’s available on AliExpress called „weten SmartHome LCD Smart Light switch“ 

€ 56,63  30％ Off | LCD Wifi Smart Wall Switch Work with Apple Homekit Alexa Google Home Smart Light Switch US 110V 220V Power Consumption
https://a.aliexpress.com/_BT6HMK

It’s basically a touch-display, some relays and some kind of Microcontroller with WiFi. 

Does anyone already have one? 
does it contain some espressif Hardware, compatible with tasmota or some other cloud-free firmware?

**Comment by barbudor** [comment]
@michaelschort ,

Yes @blakadder is looking into it with another brand : https://blakadder.com/lanbon-L8/

There is another another project also running on it : https://github.com/fvanroie/hasp-lvgl

**Comment by blakadder** [comment, marked as answer]
Tasmota runs on the chip, but screen and touch don't have drivers for it
